Gruntfile.js

```diff
@@ -27,13 +27,25 @@ module.exports = function (grunt) {
             }
         },
         jsonlint: {
-            all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json", "demos/**/*.json"]
+            all: ["package.json", ".jshintrc", "src/js/*.json", "tests/**/*.json", "demos/**/*.json"]
```

Comment:
Just an update to say this all appears to have been a problem with my own npm install on my work PC mangling things (in a way I don't quite understand, but have now fixed through the nuclear option of uninstalling and reinstalling node/npm). I apologize for indicting flocking.